MonoGame.Framework/DrawableGameComponent.cs

```diff
@@ -25,8 +25,7 @@ public int DrawOrder
                 if (_drawOrder != value)
                 {
                     _drawOrder = value;
-                    if (DrawOrderChanged != null)
-                        DrawOrderChanged(this, null);
+                    EventHelper.Raise(this, DrawOrderChanged, null);
                     OnDrawOrderChanged(this, null);
```

Comment:
Good call.  The docs clearly state that OnDrawOrderChanged raises the event.  I'll try to find more of these instances as I do my more thorough pass through to make sure events are raised using this pattern.  It might be a little while before I get around to it, but I'll keep this in mind.